### ISSUE TITLE
RangeSelector - WCAG 2.5.8 target size

### ISSUE THREAD

**Issue by britt6612** [grommet/grommet#7669]
### Is this something you or your team is interested in contributing a pull request towards?

Yes

### Expected Behavior

RangeSelector fails 2.5.8

The circular buttons on either side of the slider do not meet the minimum interactive target size of 24×24. 

once PR is merged need to update DS site.

- [ ] Update DataFilters, DataFilter rule 2.5.8

### Actual Behavior

The circular buttons are 12×12.

### URL, Screenshot, or CodeSandbox

rangeselector component

**Comment by taysea** [comment]
Noting that once this PR merges, a PR should also be filed to grommet-theme-hpe.

**Comment by britt6612** [comment]
re-opening for PR to be filed on grommet-theme-hpe
